layout/includes/comments/vssue.pug

```diff
@@ -0,0 +1,40 @@
+if theme.vssue && theme.vssue.enable
+  #vssue-container
+  link(rel="stylesheet" href="https://cdn.jsdelivr.net/npm/vssue@1.4.6/dist/vssue.css")
+  script(src="https://cdn.jsdelivr.net/npm/vue@2.6.12/dist/vue.min.js")
+  - var platform = theme.vssue.platform || 'github'
+  script(src="https://cdn.jsdelivr.net/npm/vssue@1.4.6/dist/vssue." + platform + ".min.js")
+  //- Used for scripts
+  - var perPage = theme.vssue.perPage || 10
+  - var autoCreateIssue = theme.vssue.autoCreateIssue || false
+  if theme.vssue.baseURL != ""
+    - var baseURL = theme.vssue.baseURL
+  else if ['github-v4', 'github'].includes(theme.vssue.platform)
+    - var baseURL = 'https://github.com'
+  else if theme.vssue.platform = 'gitlab'
```

Comment:
use `===` not `=`, `=` is assignment, not comparison.

Reply:
not `theme.vssue.platform = 'gitlab'` but `theme.vssue.platform === 'gitlab'`

Reply:
I forgot to fix this. Sorry